### PR TITLE
upgrade to bootstrap 3.3.4 + renaming login/out

### PR DESCRIPTION
**Pull request by Ada-Lungu**

### CODE REVIEW

File: zeeguu/account/views.py [hunk 1]
```diff
@@ -64,10 +64,11 @@ def create_account():
         password = form.get("password", None)
         email = form.get("email", None)
         name = form.get("name", None)
+        hobby = form.get("hobby", None)
```
Comment (mircealungu) [inline zeeguu/account/views.py]: Adu, can we remove the hobby code pls? It seems that it is still in there!

File: zeeguu/static/css/contributions.sass [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/* styles for contributions page */
+
+.word-action-buttons-td
```
Comment (mircealungu) [inline zeeguu/static/css/contributions.sass]: wow! you've refactored everything!! you rock!

File: zeeguu/templates/create_account.html [hunk 3]
```diff
@@ -1,74 +1,77 @@
...
+
+                    <div class="input-group-lg" >
+                        <label class="control-label">The language you are learning</label>
+                            <select autofocus id="language" required name="language" class="form-control input-lg">
+                                {% for l in languages %}
+                                    <option value="{{l.id}}">{{l.name}}</option>
+                                {% endfor %}
+                            </select>
+                    </div>
+
```
Comment (mircealungu) [inline zeeguu/templates/create_account.html]: remove this empty line pls

File: zeeguu/templates/create_account.html [hunk 4]
```diff
@@ -1,74 +1,77 @@
...
+                        <label class="control-label">Your base language</label>
+                            <select autofocus id="native_language" required name="native_language" class="form-control input-lg">
+                                {% for l in native_languages %}
+                                    <option value="{{l.id}}">{{l.name}}</option>
+                                {% endfor %}
+                            </select>
+                    </div>
+
+
+                      {% if flashed_messages %}
```
Comment (mircealungu) [inline zeeguu/templates/create_account.html]: alignment is one or two spaces off starting with here. Can we fix it?

File: zeeguu/templates/layout.html [hunk 5]
```diff
@@ -40,14 +40,13 @@
...
-            <div class="container-fluid col-md-offset-1">
-
+            <div class="container-fluid">
                 <!-- The Menu Options on the Left -->
                 <ul class="nav navbar-nav">
 
                     <!-- The Icon -->
                     {% if tab != "home" %}
-                        <li><a href="{{ url_for('account.my_account') }}"><img id="zeeguu-logo" src="../static/img/elephlogo-alone.png" alt="ZeeGuu" /></a></li>
+                            <li><a href="{{ url_for('account.my_account') }}"><img id="zeeguu-logo" src="../static/img/elephlogo-alone.png" alt="ZeeGuu" /></a></li>
```
Comment (mircealungu) [inline zeeguu/templates/layout.html]: extra tab here. is it needed?

File: zeeguu/templates/layout.html [hunk 6]
```diff
@@ -67,15 +66,14 @@
...
-                                <li><a href="{{ url_for('gym.logout') }}">Logout</a></li>
+                                <li><a href="{{ url_for('gym.logout') }}">Sign Out</a></li>
                             </ul>
                         </li>
-                    {% elif tab != "login"  %}
-                        <!-- Don't show login on the login page :) -->
-                        <li id="login"><a href="{{ url_for('gym.login') }}">LOGIN</a></li>
+                    {% elif tab != "login" and tab != "create_account" %}
+                        <!-- Don't show login on the login and create_account page :) -->
+                        <li id="login"><a href="{{ url_for('gym.login') }}">SIGN IN</a></li>
```
Comment (mircealungu) [inline zeeguu/templates/layout.html]: Make this "Sign In" instead of "SIGN IN" pls.

File: zeeguu/account/views.py (hunk 1 above)
Comment (Ada-Lungu) [inline zeeguu/account/views.py]: Done

File: zeeguu/templates/create_account.html (hunk 4 above)
Comment (Ada-Lungu) [inline zeeguu/templates/create_account.html]: Fixed

**Pull request merged**